src/main/java/org/jabref/gui/BasePanel.java

```diff
@@ -387,8 +386,8 @@ public void init() {
 
                 if (entries.isEmpty()) { // None selected. Inform the user to select entries first.
                     dialogService.showWarningDialogAndWait(
-                            Localization.lang("Autogenerate BibTeX keys"),
-                            Localization.lang("First select the entries you want keys to be generated for."));
+                    Localization.lang("Autogenerate BibTeX keys"),
```

Comment:
I think, there is again something wrong with the eclipse code style since almost all formatting changes in this file feel wrong.

Reply:
Yeah. I know. I am still figuring out how to get it right:
I am currently tweaking the behaviour a bit to not be so agressive
- Default indentation factor: I can only control how many spaces the new line will be indented, based on the Default space indentation (4 spaces)

Reply:
I think, I've found a solution: https://www.jetbrains.com/idea/download/ 😈 .